app/utils/Assets.scala

```diff
@@ -0,0 +1,30 @@
+package utils
+
+import java.net.URL
+import controllers.CachedAssets.hashedPaths
+import play.api.Play
+import org.apache.commons.io.IOUtils
+
+object Assets {
+
+  def inlineResource(path: String) = {
+    val url = AssetFinder("public/" + hashedPaths.getOrElse(path, path))
+    IOUtils.toString(url)
```

Comment:
Is there some magic going on which I'm unaware of? It auto-completed and compiled, so *seemed* to be available? But there were a couple of possibilities  shown for IOUtils